packages/discord.js/src/util/Components.js

```diff
@@ -153,5 +153,5 @@ const UserSelectMenuComponent = require('../structures/UserSelectMenuComponent')
 
 /**
  * @external JSONEncodable
- * @see {@link https://discord.js.org/#/docs/builders/main/typedef/JSONEncodable}
+ * @see {@link https://discord.js.org/docs/packages/builders/main/RawFile:Interface}
```

Comment:
This is wrong, should link to the correct function.

Reply:
This doesn't look like it exists on the newer documentation, similar to `SelectMenuBuilder`. Or am I overlooking something?

Reply:
It's at https://discord.js.org/docs/packages/util/main/JSONEncodable:Interface.